woocommerce/woocommerce-gutenberg-products-block
# Refactor Render Parent Block Function for Atomic Blocks

This is a change pulled from https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4268/files to add more flexibility to the `renderParentBlock` function. I will add inline notes regarding the specific changes.

This also converts the function to Typescript.

### How to test the changes in this Pull Request:

1. Insert the Single Product Block on a page
2. Choose a product
3. Check it renders correctly on the frontend
